src/app/queries/mocks/plans.mock.ts

```diff
@@ -231,7 +231,7 @@ const plan4: IPlan = {
   },
 };
 
-export const MOCK_PLANS: IPlan[] = [plan1, plan2, plan3, plan4];
+export const MOCK_PLANS: IPlan[] = []; //[plan1, plan2, plan3, plan4];
```

Comment:
Same here, I assume this was to test the empty state, can we revert it?

Reply:
Same!